### ISSUE TITLE
lr is very small??

### ISSUE THREAD

**Issue by dragen1860** [SwinTransformer/Video-Swin-Transformer#50]
Dear all:
  during training on video-swin, i noticed the learning rate printed is very small, say `9.6e-5`. 
```
2022-02-11 11:25:41,507 - mmaction - INFO - Epoch [8][9220/11857]	lr: 9.668e-05, eta: 4 days, 12:07:34, time: 0.665, data_time: 0.000, memory: 20882, top1_acc: 0.6312, top5_acc: 0.8387, loss_cls: 1.5623, loss: 1.5623
```

however, i find that in the swin base config file, the lr is set to `1e-3`. So is this normal?? thank you.

**Comment by Viditagarwal7479** [comment]
I think so this is normal as ```lr: 9.668e-05``` is at the 8th Epoch. Instead of ```1e-3``` as specified in the config file probably due to the ```lr_scheduler```.
